cmd/podman/kube/kube.go

```diff
@@ -0,0 +1,24 @@
+package pods
+
+import (
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/validate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Command: podman _kube1_
+	kubeCmd = &cobra.Command{
+		Use:     "kube",
+		Aliases: []string{"play"},
```

Comment:
This command is not an alias for play. Since podman kube systemd does not make sense.

Reply:
It's the other way around. play is an alias for this command, so it will still be podman play kube.